src/styles/globalStyles.ts

```diff
@@ -374,6 +374,49 @@ export const GlobalStyle = createGlobalStyle`
     line-height: 16px;
     letter-spacing: 0.05em;
   }
+
+  /* Steps */
+
+  .ant-steps-item-active {
+    .ant-steps-item-container {
+      .ant-steps-item-content {
+        .ant-steps-item-title {
+          color: ${Colors.indigo400}; 
```

Comment:
Isn't it controlled by props or LESS? Similar for below.

Reply:
It is not. Maybe I am missing something, you can check it here https://github.com/ant-design/ant-design/blob/4.x-stable/components/style/themes/default.less
I always try to resolve it with Less, if it is possible